docs/use-in-web-environment.md

```diff
@@ -0,0 +1,408 @@
+in the `<head>` section. In order to specify which meta tags to generate in the views, two methods are provided:
+
+```php
+// Creates a meta tag from an array of attributes:
+$view->registerMeta(['name' => 'keywords', 'content' => 'yii, framework, php']);
+
+// Creates a meta tag from a `Yiisoft\Html\Tag\Meta` instance:
+$view->registerMetaTag(\Yiisoft\Html\Html::meta()->name('robots')->content('noindex'));
+```
+
+The above code will register a meta tags in the `Yiisoft\View\WebView` instance. The registered meta tag is rendered
+after the view template finishes rendering. The following HTML code will be generated and inserted at the place
+where you call `Yiisoft\View\WebView::head()` in the view template:
+
+```html
+<meta name="keywords" content="yii, framework, php">
+<meta name="robots" content="noindex">
+```
+
+Note that if you call `registerMeta()` or `registerMetaTag()` multiple times, it will register multiple meta tags,
+regardless whether the meta tags are the same or not. To make sure there is only a single instance of a meta tag type,
+you can specify a key as a second parameter when calling the methods. For example, the following code registers two
+`description` meta tags. However, only the second one will be rendered.
+
+```php
+$view->registerMeta(
+    ['name' => 'description', 'content' => 'This is my cool website made with Yii!'],
+    'description',
+);
+
+$view->registerMetaTag(
+    \Yiisoft\Html\Html::meta()->name('description')->content('This website is about funny raccoons.'),
+    'description',
+);
+```
+
+## Registering link tags
+
+Like [meta tags](#registering-meta-tags), link tags are useful in many cases, such as customizing favicon, pointing to
+RSS feed or delegating OpenID to another server. You can work with link tags in the similar way as meta tags by using
+`registerLink()` or `registerLinkTag()`. For example, in a content view, you can register a link tag like follows:
+
+```php
+// Creates a link tag from an array of attributes:
+$view->registerLink([
+    'title' => 'Live News for Yii',
+    'rel' => 'alternate',
+    'type' => 'application/rss+xml',
+    'href' => 'https://www.yiiframework.com/rss.xml',
+]);
+
+// Creates a link tag from a `Yiisoft\Html\Tag\Link` instance:
+$view->registerLinkTag(\Yiisoft\Html\Html::link('/myicon.png', [
+    'rel' => 'icon',
+    'type' => 'image/png',
+]));
+```
+
+The code above will result in:
+
+```html
+<link title="Live News for Yii" rel="alternate" type="application/rss+xml" href="http://www.yiiframework.com/rss.xml">
+<link rel="icon" type="image/png">
+```
+
+You can use the second parameter to pass the position at which the link tag should be inserted in a page,
+default by `Yiisoft\View\WebView::POSITION_HEAD`. Similar to registering meta tags, you can specify
+a key to avoid creating duplicate link tags. Only in the `registerLink()` and `registerLinkTag()`
+methods the key is specified by the third parameter.
```

Comment:
```suggestion
You can use second parameter to specify position at which the link tag should be inserted in a page.
Default is `Yiisoft\View\WebView::POSITION_HEAD`. Similar to registering meta tags, you can specify
a key to avoid creating duplicate link tags. In the `registerLink()` and `registerLinkTag()` the key is specified as a third parameter.
```